.github/workflows/flowzone.yml

```diff
@@ -0,0 +1,45 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone-staging:
+    name: Deploy to balena-staging
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
+    secrets:
+      FLOWZONE_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+      BALENA_API_KEY: ${{ secrets.BALENA_STAGING_TOKEN }}
+    with:
+      balena_environment: balena-staging.com
+      protect_branch: false
+      balena_slugs: |
+        balena_os/aarch64-supervisor,
+        balena_os/amd64-supervisor,
+        balena_os/armv7hf-supervisor,
+        balena_os/i386-supervisor,
+        balena_os/rpi-supervisor
+
+  flowzone:
+    name: Deploy to balena-cloud
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
+    secrets:
+      FLOWZONE_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+      BALENA_API_KEY: ${{ secrets.BALENA_TOKEN }}
+    with:
+      balena_environment: balena-cloud.com
+      protect_branch: false
```

Comment:
Reminder to whomever would like to merge this if it passes all checks -- remove both `protect_branch` lines. (They default to `true`)